lib/serializer.js

```diff
@@ -113,7 +113,9 @@ module.exports = class Serializer {
       }
     }
 
-    if (str.length < 42) {
+    if (format === 'raw') {
```

Comment:
I would use a different name than `raw` to emphasize that is format has a very dangerous potential security. Raw method would not escape a double quote char which makes it very easy to inject something into the data. You can use it only if you 200% sure in your data. I would call it something like `unsafe`.

@mcollina WDYT?